indico/modules/categories/client/js/components/CategoryModeration.jsx

```diff
@@ -0,0 +1,131 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2021 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import eventRequestsURL from 'indico-url:categories.api_event_move_requests';
+import eventURL from 'indico-url:events.display';
+
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import {Button, Checkbox, Image, Table} from 'semantic-ui-react';
+
+import {handleSubmitError} from 'indico/react/forms';
+import {useIndicoAxios} from 'indico/react/hooks';
+import {Translate} from 'indico/react/i18n';
+import {indicoAxios} from 'indico/utils/axios';
+import {serializeDate} from 'indico/utils/date';
+import './CategoryModeration.module.scss';
+
+function EventRequestList({requests, onApprove, onReject}) {
+  const [selected, _setSelected] = useState(new Set());
+  const isAnySelected = selected.size > 0;
+
+  const submit = event => {
+    _setSelected(new Set());
+    event(Array.from(selected));
+  };
+
+  const select = id =>
+    _setSelected(state => {
+      const set = new Set([...state, id]);
+      if (state.has(id)) {
+        set.delete(id);
+      }
+      return set;
+    });
+
+  if (requests.length === 0) {
+    return <Translate as="p">There are no event move requests</Translate>;
+  }
+
+  return (
+    <>
+      {/* TODO: check if module is enabled */}
```

Comment:
Probably not needed; for many other things we don't do it either since accessing the page via a direct link is harmless anyway as there's nothing useful to do on it.